sdk/template/template/package.json

```diff
@@ -58,7 +58,6 @@
   },
   "homepage": "https://github.com/azure/azure-sdk-for-js/tree/master/sdk/template/template",
   "sideEffects": false,
-  "private": true,
```

Comment:
Out of curiosity, are we switching to public by default now?

Reply:
No - I use this package as a canary to test that our end-to-end deployment pipeline works. It means that I can publish multiple versions of this package a day without too much confusion. Unfortunately there just isn't a good way to test end to end pipelines.